packages/eslint-plugin-jest/src/index.js

```diff
@@ -8,11 +8,33 @@
  * @flow
  */
 
-/* eslint-disable global-require */
-
 'use strict';
 
-module.exports.rules = {
-  'no-exclusive-tests': require('./rules/no-exclusive-tests'),
-  'no-identical-title': require('./rules/no-identical-title'),
+module.exports = {
+  environments: {
```

Comment:
This makes it so when using the jest eslint plugin you can get the globals from there. It allows us to update the global variables in Jest and in this plugin without having to send a PR to the "globals" project. Arguably, Jest shouldn't be a global env available through eslint anyway.